src/dscanner/main.d

```diff
@@ -155,6 +157,9 @@ else
 	if (absImportPaths.length)
 		moduleCache.addImportPaths(absImportPaths);
 
+	if (reportFormat != "")
```

Comment:
I think this comparision can have issues if reportFormat was initialized to `null`. I would do it like in the if check above and use `if (reportFormat.length)` instead

Reply:
fixed